src/tensors/tensor.jl

```diff
@@ -581,27 +581,19 @@ end
 # Real and imaginary parts
 #---------------------------
 function Base.real(t::AbstractTensorMap)
-    # `isreal` for a `Sector` returns true iff the F and R symbols are real. This guarantees
-    # that the real/imaginary part of a tensor `t` can be obtained by just taking
-    # real/imaginary part of the degeneracy data.
-    if isreal(sectortype(t))
-        return TensorMap(real(t.data), codomain(t), domain(t))
```

Comment:
So this one slipped through in the previous change, as it should actually also only work for `TensorMap` arguments. That one is not entirely straightforward to define for `AbstractTensorMap`.